bindings/xen/solo5_xen.lds

```diff
@@ -55,16 +55,17 @@ SECTIONS {
      */
     _stext = .;
 
-    .interp : {
-        *(.interp)
-    } :interp :text
-
     .text :
     {
+        *(.data.multiboot)
         *(.text)
         *(.text.*)
     } :text
 
+    .interp : {
+        *(.interp)
+    } :interp :text
+
```

Comment:
Why did this need moving, exactly? It doesn't really matter for Xen, but I'd still like to understand what's going on.

Reply:
The multiboot header needs to be in the first 8k of the file. Without this change, first 4k were program headers, then the second 4k were just this section, pushing multiboot header beyond first 8k.

Reply:
You should be able to accomplish the same thing without re-ordering by placing `*(.data.multiboot)` into the `.text` output section as is done in `solo5_virtio.lds`?

Reply:
It's already here and it wasn't enough. `.text` was linked at 8k offset in the file - just beyond what we need. I'm not sure what's the difference with virtio, perhaps more notes here?

Reply:
Ok, don't worry about it, leave it as-is (and in a separate commit). Thanks.

Reply:
(... looking at the linker scripts and readelf output, I actually think `.interp` doesn't belong in `:text` at all but I'm not going to muck with that now...)